### ISSUE TITLE
Scrobble vinyls in real time, showing them as "now playing"

### ISSUE THREAD

**Issue by elamperti** [elamperti/OpenWebScrobbler#205]
**Is your feature request related to a problem? Please describe**
Some users listen to vinyls and would like to have last.fm show what they are currently listening to as their "now playing" track. This would require some kind of player/interface to tag along while the user listens to a vinyl.

**Additional context**
Request on Discord by owen#7245 ([permalink](https://discord.com/channels/474201109980905483/474203482065207296/1093988204857806869))

**Comment by standuptall** [comment]
this could be done  allowing the app to use microphone and use an online api provider (eg. Shazam) to recognize the song.

**Comment by elamperti** [comment]
@standuptall indeed, but being a web app the options are limited at the moment (Shazam specifically doesn't have an API). There are some alternatives I'll consider whenever I get around to this though.

**Comment by garret1317** [comment]
surely you could just have a "play" button for each side that scrobbles each track as its duration passes?
start the record and press the button at ~roughly the same time, and it all syncs up
